client/tests/components/gameCard.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { describe, it, expect } from "vitest";
+import { render, screen, within } from "@testing-library/react";
+//import { userEvent } from "@testing-library/user-event";
+import React from "react";
+import GamePage from "../../src/pages/GamePage/GamePage";
+import "@testing-library/jest-dom";
+import { MemoryRouter } from "react-router-dom";
+import { socket } from "../../src/services/socketServices";
+
+//const user = userEvent.setup();
+
+const lobbyIdMock = "1234";
+const membersArrayMock = ["testUser1", "testUser2"];
+
+describe("Game Page Unit Test:", () => {
+ it("renders the page", () => {
+  render(
+   <MemoryRouter>
+    <GamePage
+     socket={socket}
+     userId={"test user 1"}
+     members={membersArrayMock}
+     lobbyId={lobbyIdMock}
+    />
+   </MemoryRouter>
+  );
+
+  const h1 = screen.getByRole("heading", {level: 1});
+  const redRow = screen.getByRole("list", { name: "row-red" });
+  const buttons = within(redRow).getAllByRole("button");
+    
+
+
+  expect(h1).toHaveTextContent(`Lobby: ${lobbyIdMock}`);
+  expect(redRow).toBeInTheDocument();
```

Comment:
Small thing, but ".toBeVisible()" is perhaps more reflective of a user's experience in this case as it asserts that the element is visible rather than only being in the DOM tree and not potentially not visible.

Reply:
I have updated code to replace ToBeInTheDocument with ToBeVisible